src/components/HomePage/Banner.js

```diff
@@ -81,41 +97,42 @@ const useStyles = makeStyles(theme => ({
 
 function renderInfo(classes) {
 	const tz = getTimeZoneWithFormat(hothStart, 'short');
-	const month = monthFormatter.format(hothStart);
+	//const month = monthFormatter.format(hothStart);
 	const startDay = hothStart.getDate();
 	const endDay = hothEnd.getDate();
 	const eventCrossesDate = startDay !== endDay;
-	const endDayString = eventCrossesDate ? `–${endDay}` : '';
+	//const endDayString = eventCrossesDate ? `–${endDay}` : '';
 	return (
 		<Grid
 			container
 			direction='column'
-			justify='center'
-			alignItems='center'
+			justify='left'
+			alignItems='left'
 		>
-			<Typography variant='h3' className={classes.text} component='h2' style={{ fontSize: '3em' }}>
-				Hack off the Hill 8
+			<img src = {hothTitle}
+			style={{ paddingLeft: 50, paddingBottom: 20, width: '80%' }}
+			/>
```

Comment:
We don't need to use an image for the text title. According to the Figma, the title is just the standard Poppins font. Additionally, you can export the gradient on the "8" in CSS format by clicking on the "Copy" button here:

![image](https://user-images.githubusercontent.com/1538624/106512175-40857d80-649f-11eb-8f2d-49d742bf723b.png)

That puts the following CSS string onto the system clipboard:
```css
background: linear-gradient(199.69deg, #FF5F96 35.64%, #BD01FF 95.25%);
```

That's not entirely enough to make the gradient work for text, but [this post](https://cssgradient.io/blog/css-gradient-text/) documents the other two properties you need to set:

```css
-webkit-background-clip: text;
-webkit-text-fill-color: transparent;
```

Check out Jamie's modifications for how it all fits together.